menas/src/test/resources/application.properties

```diff
@@ -68,7 +68,9 @@ menas.environment=localhost
 
 menas.spark.master=local[1]
 
-spline.urlTemplate=http://localhost:8080/spline/dataset/lineage/_search?path=%s&application_id=%s
+# Avoiding the http(s) protocol will cause that the same protocol will be used as for Menas itself, avoiding issues
+menas.lineage.readApiUrl=//localhost:8080/spline-gw/consumer
+lineageService=//localhost:8080/spline-ui/app/lineage-overview?executionEventId=%s
```

Comment:
I am not sure about this configuration 🤔

Reply:
It shouldn't be there.